exe/etcd-env

```diff
@@ -40,10 +43,17 @@ def fetch(uri_str, limit = 10)
 end
 
 if __FILE__ == $PROGRAM_NAME
```

Comment:
It doesn't. Both of them come with a full path which is not the same. Do we really need it? You can't anyway really require it as it is.